resources/todomvc/architecture-examples/react-redux/src/components/item.jsx

```diff
@@ -0,0 +1,62 @@
+import { PureComponent } from "react";
+import classnames from "classnames";
+import PropTypes from "prop-types";
+import TextInput from "./text-input";
+
+export default class Item extends PureComponent {
+    static propTypes = {
+        todo: PropTypes.object.isRequired,
+        editTodo: PropTypes.func.isRequired,
+        deleteTodo: PropTypes.func.isRequired,
+        toggleTodo: PropTypes.func.isRequired,
```

Comment:
I considered about connecting `Item`, but I'm not so sure... We can look at it in a follow-up possibly.